Workbooks/Network Insights/AzureFirewallWorkbooks/NetworkInsights-AzureFirewallMetrics-StructuredLogs/NetworkInsights-AzureFirewallMetricsStructuredLogs.workbook

```diff
@@ -0,0 +1,2670 @@
+{
+    "version": "Notebook/1.0",
+    "items": [
+      {
+        "type": 1,
+        "content": {
+          "json": "## Azure Firewall Workbook\r\n---\r\n"
+        },
+        "name": "text - 23"
+      },
+      {
+        "type": 11,
+        "content": {
+          "version": "LinkItem/1.0",
+          "style": "tabs",
+          "links": [
+            {
+              "id": "20847ce8-91bc-4d8b-b878-a5a41c95c31c",
+              "cellValue": "selectedTab",
+              "linkTarget": "parameter",
+              "linkLabel": "Azure Firewall Overview",
+              "subTarget": "AFOverview",
+              "preText": "Azure Firewall Overview",
+              "style": "link"
+            },
+            {
+              "id": "f564709d-1658-46a1-8b44-892210e13017",
+              "cellValue": "selectedTab",
+              "linkTarget": "parameter",
+              "linkLabel": "Azure Firewall - Application rule log statistics",
```

Comment:
still not screenshots to verify but this seems like a really long title for tabs.

"Azure Firewall - Application Rule log statistics" is the the title of a tab?

does it need azure firewall at all?

Reply:
This is how the tabs look like today:

![image](https://github.com/microsoft/Application-Insights-Workbooks/assets/44907284/287caf91-e869-46fe-a7ff-914146bccff7)

I will discuss the tab names change with my team, but I don't want to have a dependency on this to publish the new workbook, since the other 2 workbooks are like that already. Is that ok?

Reply:
unclear what that means.  why publish something subpar because something else is subpar?  i don't understand why the word "Azure Firewall" needs to appear 5 times at the top of this template.  on narrow monitors, all those tabs are going to end up in the "..." overflow (some already are?) and users won't even notice they are there.

Reply:
Removed multiple references to Azure Firewall from the workbook gallery file

Reply:
<img width="1641" alt="image" src="https://github.com/microsoft/Application-Insights-Workbooks/assets/44907284/61d1ff96-702f-42d1-847f-ef26171629e2">

Reply:
Hey @gardnerjr, do you have any additional comments to share? Is the PR good to be approved?

Reply:
@gumoden i've only been reviewing for technical things, where you were using the wrong formats of files, or general suggestions.

the network insights team is really the codeowners of this area, someone from that team really should be approving since they'd be generally responsible for it.